filters/f_auto_filters.c

```diff
@@ -73,27 +73,47 @@ static void deint_process(struct mp_filter *f)
         return;
     }
 
+    char *field_parity;
+    switch (opts->field_parity) {
+    case 0: {
+        field_parity = "tff"; 
+        break;
+    }
+    case 1: {
+        field_parity = "bff";
+        break;
+    }
+    default: {
+        field_parity = "auto";
+    }
+    }
+
```

Comment:
We do not use braces around case blocks unless they are required for local variable declarations.